[Update 2.5] Add mention of powered by header and how to configure it (#1223)
ezsystems/developer-documentation: docs/updating/4_update_2.5.md

    CREATE INDEX ezpage_pages_content_id_version_no ON ezpage_pages(content_id, version_no);
    ```

## Updating to 2.5.15

> ### Powered By Header

Reviewer: ```suggestion
### Powered-By header
```

> In order to be able to promote use of eZ Platform, `ezsystems/ez-support-tools` v1.0.9 adds capability to set powered by header.

Reviewer: ```suggestion
In order to promote use of eZ Platform, `ezsystems/ez-support-tools` v1.0.9 sets the Powered-By header.
```
Or maybe, as we shouldn't expect users know which package has which version in which release:
```suggestion
In order to promote use of eZ Platform, `ezsystems/ez-support-tools` as of eZ Platform v2.5.15 sets the Powered-By header.
```

> By default this is enabed, and will generate a header like `Powereed-By: eZ Platform Enterprise 2.5`.

Reviewer: ```suggestion
It is enabled by default and generates a header like `Powered-By: eZ Platform Enterprise 2.5`.
```

> If you'd like to ommit version number, for instance if you are stuck on a fast track release beyond it's maintenance window:

Reviewer: ```suggestion
To omit the version number, for instance if you are stuck on a fast track release beyond its maintenance window,
use the following configuration:
```

``` yaml
ezplatform_support_tools:
    system_info:
        powered_by:
            release: "none"
```

> If you however want to ommit the whole feature, you can likewise do that by disabling it:

Reviewer: ```suggestion
To opt out of the whole feature, by disabling it with the following configuration:

```
